src/lib/tracker.js

```diff
@@ -75,3 +75,11 @@ export async function aliasUser( vipUserId ): Promise<Response> {
 export async function trackEventWithEnv( appId, envId, eventName, eventProps = {} ): Promise<Response> {
 	return trackEvent( eventName, { ...eventProps, app_id: appId, env_id: envId } );
 }
+
+export function makeCommandTracker( command, trackingInfo = {} ) {
```

Comment:
Have you done this so that you can tag the trackers at different (seemingly unrelated) files with the same command?

Reply:
We need to pass the function to [command classes](https://github.com/Automattic/vip-cli/tree/develop/src/commands) because the same class can be used in multiple commands (for example, [this](https://github.com/Automattic/vip-cli/blob/develop/src/commands/dev-env-import-sql.js) one is used in both dev-env-import-sql and dev-env-sync-sql commands. 

I thought creating a command-tracker will help us keep things streamlined a bit for the future commands.